app/src/main/java/io/skygear/chatdemo/adapter/ChatUsersAdapter.java

```diff
@@ -57,6 +57,24 @@ public int getItemCount() {
         return new ViewHolder(layoutInflater.inflate(R.layout.item_chat_user, parent, false));
     }
 
+    private void refreshChatUserBackgroundColor(ChatUser chatUser, TextView textView) {
+        refreshChatUserBackgroundColor(chatUser, textView, null);
+    }
+
+    private void refreshChatUserBackgroundColor(ChatUser chatUser, View view, Boolean alreadySelected) {
+        alreadySelected = alreadySelected == null ?
+                            isChatUserSelected(chatUser) :
+                            alreadySelected;
+        int backgroundColor = alreadySelected ?
+                ContextCompat.getColor(context, R.color.transparent) :
+                ContextCompat.getColor(context, R.color.gray_light);
+        view.setBackgroundColor(backgroundColor);
+    }
+
+    private boolean isChatUserSelected(ChatUser chatUser) {
+        return mSelectedChatUsers.indexOf(chatUser) == -1;
```

Comment:
The function name is not correct, `mSelectedChatUsers.indexOf(chatUser) == -1` means the user is not selected